libsodium-sys/libsodium

```diff
@@ -0,0 +1 @@
+Subproject commit f01d7c128214e8f976889c8093a42d9cae1abe4b
```

Comment:
Ahh, I didn't know about what was coming in 1.0.11. Nifty!

I'd say from the conservative perspective, using specific stable releases whenever possible in cryptography libraries makes me do less of a double-take though (and it looks like the author signs releases, but not individual commits). I suppose this will be a non-issue once 1.0.11 is released officially.